spec/features/brexit_checker/question_results_spec.rb

```diff
@@ -27,10 +41,33 @@
     then_i_should_see_the_no_results_page
   end
 
+  def and_i_should_see_the_citizens_action_header
+    expect(page).to have_content I18n.t!("brexit_checker.results.audiences.citizen.heading")
```

Comment:
I would normally check that the text is contained within a specific element, either by class or (for example) a h2. Otherwise it could be somewhere else on the page and your test would still pass.